cmd/podman/common.go

```diff
@@ -500,3 +500,19 @@ func scrubServer(server string) string {
 	server = strings.TrimPrefix(server, "https://")
 	return strings.TrimPrefix(server, "http://")
 }
+
+func UsageTemplate() string {
```

Comment:
I think lint might complain here because it's an uncommented public function

Reply:
Doesn't appear to complain, but it would be good to have a little doc explaining what this function does.

Reply:
@baude Other commit is going to conflict with this and might make this a lot easier to implement.  Wait until his commit gets in.